WordPress/Classes/Services/JetpackNotificationMigrationService.swift

```diff
@@ -47,13 +70,8 @@ final class NotificationFilteringService {
         return allowDisablingWPNotifications && isWordPress && notificationsEnabled
     }
 
-    func disableWordPressNotificationsIfNeeded() {
-        if allowDisablingWPNotifications, !isWordPress {
-            wordPressNotificationsEnabled = false
-        }
-    }
 
-    func shouldFilterWordPressNotifications() -> Bool {
+    func shouldDisableWordPressNotifications() -> Bool {
```

Comment:
```suggestion
    func shouldDisableNotifications() -> Bool {
```
Since this can be called in the context of either the WP or JP app, I think it  could be renamed.

Reply:
Agree, good point.

Reply:
Marking resolved